site/reference/fn/eval/README.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "`eval`"
+linkTitle: "eval"
+type: docs
+description: >
+  Execute function on resources
+---
+
+<!--mdtogo:Short
+    Execute function on resources
+-->
+
+`eval` executes function on resources in a directory and writes the output to
+the local filesystem in-place. Functions are packaged as container images or
```

Comment:
Not only in-place? It can support out-place with `--dry-run` or `fn sink`

Reply:
Removed the `in-place` bit.